AutomatedTesting/Gem/PythonTests/Physics/tests/force_region/ForceRegion_LinearDampingForceOnRigidBodies.py

```diff
@@ -161,14 +161,42 @@ def __init__(self, name):
 
     sphere.initial_velocity = azlmbr.physics.RigidBodyRequestBus(azlmbr.bus.Event, "GetLinearVelocity", sphere.id)
 
-    level_correct = (
-        (abs(sphere.initial_pos.y - force_region.initial_pos.y) < CLOSE_ENOUGH)
-        and (abs(sphere.initial_pos.y - trigger.initial_pos.y) < CLOSE_ENOUGH)
-        and (abs(sphere.initial_pos.x - force_region.initial_pos.x) < CLOSE_ENOUGH)
-        and (abs(sphere.initial_pos.x - trigger.initial_pos.x) < CLOSE_ENOUGH)
-        and (sphere.initial_pos.z > force_region.initial_pos.z > trigger.initial_pos.z)
-        and sphere.initial_velocity.IsClose(INITIAL_VELOCITY, CLOSE_ENOUGH)
-    )
+    # Starting Level State Values
+    Report.info(f"Sphere Initial Values - X: {sphere.initial_pos.x} Y: {sphere.initial_pos.y} Z: {sphere.initial_pos.x} Velocity: {sphere.initial_velocity}")
+    Report.info(f"Force Region Initial Values - X: {force_region.initial_pos.x} Y: {force_region.initial_pos.y} Z: {force_region.initial_pos.z}")
+    Report.info(f"Trigger Initial Values - X: {trigger.initial_pos.x} Y: {trigger.initial_pos.y} Z: {trigger.initial_pos.z}")
+
+    # Perform Calculations
+    sphere_acceleration_y = abs(sphere.initial_pos.y - force_region.initial_pos.y)
+    sphere_damping_y = abs(sphere.initial_pos.y - trigger.initial_pos.y)
+    sphere_acceleration_x = abs(sphere.initial_pos.x - force_region.initial_pos.x)
+    sphere_damping_x = abs(sphere.initial_pos.x - trigger.initial_pos.x)
+
+    # Perform Checks
+    sphere_acceleration_y_valid = sphere_acceleration_y < CLOSE_ENOUGH
+    sphere_damping_y_valid = sphere_damping_y < CLOSE_ENOUGH
+    sphere_acceleration_x_valid = sphere_acceleration_x < CLOSE_ENOUGH
+    sphere_damping_x_valid = sphere_damping_x < CLOSE_ENOUGH
+    setup_entity_priority_valid = sphere.initial_pos.z > force_region.initial_pos.z > trigger.initial_pos.z
+    sphere_is_close = sphere.initial_velocity.IsClose(INITIAL_VELOCITY, CLOSE_ENOUGH)
+    level_correct = (sphere_acceleration_y_valid and sphere_damping_y_valid and sphere_acceleration_x_valid
+                     and sphere_damping_x_valid and setup_entity_priority_valid and sphere_is_close)
+
+    # Report
+    if not sphere_acceleration_y_valid:
+        Report.info(f"Sphere Acceleration Y was not within expected bounds.")
```

Comment:
From this error message it's not clear that the sphere acceleration comes from comparing sphere and force_region initial positions.

I think it'd be much clearer if each message would be explicit about what values don't match, instead of printing first all the variables and then from a message having to figure out which one it's comparing.

````
if not sphere_damping_y_valid:
        Report.info(f"Sphere initial Y position {sphere.initial_pos.y} is not close to force region initial Y position {force_region.initial_pos.y}.")
...
````